index.html

```diff
@@ -8,250 +8,288 @@
+        </li>
+        <li class="tech-list">
+          <span>
+            <img class="list-type-img" src="images/AboutMe/Square.svg" alt=""> Frameworks
+          </span>
             <ul class="no-decoration dropdown-list">
-              <li>
-                <a href="#" class="text-white no-decoration">
-                  Database Management
-                </a>
-              </li>
-              <li>
-                <a href="#" class="text-white no-decoration">
-                  Version Control
-                </a>
-              </li>
-              <li>
-                <a href="#" class="text-white no-decoration">
-                  CLI
-                </a>
-              </li>
-              <li>
-                <a href="#" class="text-white no-decoration">
-                  API Design
-                </a>
-              </li>
-              <li>
-                <a href="#" class="text-white no-decoration">
-                  Web Development
-                </a>
-              </li>
-            </ul>
-          </li>
-        </ul>
-      </div>      
-    </section>
+            <li>
+              <a href="#" class="text-white no-decoration">
+                React.js
+              </a>
+            </li>
+            <li>
+              <a href="#" class="text-white no-decoration">
+                Ruby on Rails
+              </a>
+            </li>
+            <li>
+              <a href="#" class="text-white no-decoration">
+                RSpec
+              </a>
+            </li>
+            <li>
+              <a href="#" class="text-white no-decoration">
+                Capybara
+              </a>
+            </li>
+          </ul>
+        </li>
+        <li class="tech-list">
+          <span>
+            <img class="list-type-img" src="images/AboutMe/Disc.svg" alt="">
+            Skills
+          </span>            
+          <ul class="no-decoration dropdown-list">
+            <li>
+              <a href="#" class="text-white no-decoration">
+                Database Management
+              </a>
+            </li>
+            <li>
+              <a href="#" class="text-white no-decoration">
+                Version Control
+              </a>
+            </li>
+            <li>
+              <a href="#" class="text-white no-decoration">
+                CLI
+              </a>
+            </li>
+            <li>
+              <a href="#" class="text-white no-decoration">
+                API Design
+              </a>
+            </li>
+            <li>
+              <a href="#" class="text-white no-decoration">
+                Web Development
+              </a>
+            </li>
+          </ul>
+        </li>
+      </ul>
+    </div>      
+  </section>
+  <!-- Contact Form Starts -->
+  <section class="section-contact-form">
+    <h1 class="text-dark">
+      I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+    </h1>
+    <form>
```

Comment:
- [ ] Kindly note that you need to add the action and method attributes to the form tag so that the data would be posted and you will receive it in your mail. To do this, you will need to:
- Add `action="https://formspree.io/f/{form_id}"` To get the form id, you will need to create an account with formspree.io
- Add `method="POST"` to the form tag too so it would look like this:
```
<form method="POST" action="https://formspree.io/f/{form_id}">
```

Reply:
@Olamarx Thank you so much.